README.md

```diff
@@ -52,50 +43,38 @@ _NOTE:_
+| Intel® Emerald Rapids | Ubuntu 23.10 | Setup TDX host referring to [here](https://github.com/canonical/tdx) | Ubuntu 22.04 | Build packages referring to [here](https://github.com/intel/tdx-tools/tree/tdx-1.5/build/ubuntu-22.04)| Setup containerized [PCCS](https://github.com/intel/confidential-cloud-native-primitives/tree/main/container/pccs) and [QGS](https://github.com/intel/confidential-cloud-native-primitives/tree/main/container/qgs) on the host | 
 
-![](docs/ccnp_arch.png)
+_NOTE: the following installation will be performed in a confidential VM. Make sure you have confidential VM booted before moving forward._
 
-*Note: For Intel TDX, it bases on Linux TDX Software Stack at [tdx-tools](https://github.com/intel/tdx-tools), the corresponding white
-paper is at [Whitepaper: Linux* Stacks for Intel® Trust Domain Extension 1.0](https://www.intel.com/content/www/us/en/content-details/779108/whitepaper-linux-stacks-for-intel-trust-domain-extension-1-0.html).*
+### 2.2 Deploy CCNP Services
 
+It supports to deploy CCNP services as DaemonSets in Kubernetes cluster or docker containers on a single confidential VM. Please refer to below guides for different deployment environments.
 
-## 3. Installation
+- [CCNP deployment guide - K8S](deployment/README.md): on confidential VM node of Kubernetes cluster.
 
-Here provides the description on the installation steps for the services and the
-SDK.
+- [CCNP deployment guide - Docker](deployment/README.md): on confidential VM using docker compose.
 
-[CCNP deployment guide](deployment/README.md) introduces how to deploy CCNP services,
-which also includes an example of running CCNP example pod to get cloud native primitives
-using CCNP SDK.
+This project should also be able deployed on [diverse cloud native PaaS frameworks](https://www.redhat.com/en/blog/confidential-computing-use-cases) like confidential cluster, `kubevirt` etc in future.
+An example of landing architecture on confidential cluster is as follows.
 
-You can also use the following guides as alternatives of installing each service
-separately.
+![](/docs/ccnp-landing-confidential-cluster.png)
 
-- Quote Server: [Installation guide](service/quote-server/README.md)
-- Measurement Server: [Installation guide](service/measurement-server/README.md)
-- Event Log Server: [Installation guide](service/eventlog-server/README.md)
+### 2.3 Install SDK
 
-For SDK, user can simply install from PyPI using command:
+CCNP SDK can be used by workload for cloud native primitives collecting. It needs to be installed within the workload container image and called whenever the primitives are required. The SDK can be installed from PyPI using command:
```

Comment:
"used by a workload"

Reply:
"The SDK can be installed from PyPi using the command:"

Reply:
For the next section I would indicate that the repository needs to be cloned into the VM if you want to install that way.  I would rephrase as  "Alternatively, the CCNP can be installed from source code with the following command. Make sure to clone the repository into your confidential VM."